.github/workflows/release.yml

```diff
@@ -16,134 +16,206 @@ on:
 # Upgrade to gcc-11 to prevent it from using its builtins (#14150)
 jobs:
   linux:
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
     runs-on: ubuntu-18.04
     outputs:
-      version: ${{ steps.build.outputs.version }}
-      release: ${{ steps.build.outputs.release }}
+      # this metadata is used when constructing the GH release post
+      version: ${{ steps.metadata.outputs.version }}
+      release: ${{ steps.metadata.outputs.release }}
     steps:
-      - uses: actions/checkout@v2
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
         with:
-          fetch-depth: 0
-      - name: Install dependencies
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y autoconf automake build-essential cmake gcc-11 gettext gperf libtool-bin locales ninja-build pkg-config unzip
-      - if: github.event_name == 'push' || (github.event_name == 'workflow_dispatch' && github.event.inputs.tag_name != 'nightly')
+          image: tonistiigi/binfmt:latest
+          platforms: arm64 # Platforms to install (arm64,riscv64,arm ; default all)
+      -
+        name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true # build -> buildx automatically
+      -
+        if: github.event_name == 'push' || (github.event_name == 'workflow_dispatch' && github.event.inputs.tag_name != 'nightly')
         run: printf 'NVIM_BUILD_TYPE=Release\n' >> $GITHUB_ENV
-      - if: github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.tag_name == 'nightly')
+      -
+        if: github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.tag_name == 'nightly')
         run: printf 'NVIM_BUILD_TYPE=RelWithDebInfo\n' >> $GITHUB_ENV
-      - name: Build release
-        id: build
+      -
+        name: Build Neovim and Create .tar.gz for ${{ matrix.arch }}
+        run: |
+          docker build --platform linux/${{ matrix.arch }} \
+                 --build-arg CC=gcc-11 \
+                 --build-arg CMAKE_BUILD_TYPE=${NVIM_BUILD_TYPE} \
+                 --build-arg CMAKE_EXTRA_FLAGS="-DCMAKE_INSTALL_PREFIX:PATH=" \
+                 --build-arg ARCH=${{ matrix.arch }} \
+                 -f .github/workflows/dockerfile-output-targz \
+                 -o $GITHUB_WORKSPACE/build .
+      -
+        name: Output version and release for GH release notes
```

Comment:
Would enjoy an `if arch == amd64` since we can't actually run the arm bin and only need to run *something* once. Currently it "harmlessly errors" for arm.